### ISSUE TITLE
Further improve pool filtering

### ISSUE THREAD

**Issue by iamvukasin** [tempus-finance/tempus-app#521]
I've started this topic by submitting #516 and I now have a few suggestions that may improve the UX of filtering even more:

- [ ] Asset and protocol fields could be replaced with drop-down selections, so users can choose from the options available in the pools.
- [ ] Maturity could indicate that the expected values for min and max are in days. Date pickers would fit perfectly here since `Date` is used in the underlying filtering code anyway.

I guess `TokenSelector` would solve the first issue. However, its design is different compared to the rest of the filtering fields, so either `TokenSelector` should be adapted to the others or all the fields would be changed to match the other components in the rest of the app.

If you agree with these suggestions, I'll be glad to help to complete them.

**Comment by lazarmitic** [comment]
Idea to add drop-down for token and protocol is a good one, we have it our backlog but didn't have time to do it. 

We definitely need to indicate that maturity inputs are in days, some users already complained about this. We did have date pickers in 1st iteration, and then we switched to day inputs.

Both of these are design/UX related issues, and our designer is currently overloaded with work. As soon as he have some free time we can discuss changes you suggested and let you know what are next steps.
